### ISSUE TITLE
Q4.2

### ISSUE THREAD

**Issue by vahidAK** [STAT540-UBC/Discussion#321]
Hi,
Should we define a contrast matrix for q4.2 or should we left the baseline as default which I think limma considers E16 as the baseline?
Thanks.

**Comment by janxue** [comment]
>**Question 4.2**
>Use limma-voom to fit a linear model with cell type, organism part, age and the interaction between age and cell type as covariates (hint: use lmFit, voom and eBayes). Before you do this, reformat the data frame so that gene IDs are row names, and not a column (limma requires the dataset in this format).

I don’t think you would want to make a contrast matrix. Based on the question it doesn’t logically seem like you would want to be contrasting here, maybe I’m wrong and the TA’s could correct this.

Would you possibly mean fitting a linear model with *cell means parameterization* rather than the default *reference-treatment effect parameterization* r does when producing the design matrix?

I left it as the default *reference-treatment effect parameterization* but I didn’t have a particularly good reason for this, I just thought that it would allow me to find differentially expressed genes anyway.

To follow up with this question, to the TA’s, would you be able to give an intuitive sense of when it would be better to use one parameterization over the other (i.e. cell means vs reference treatment)?

**Comment by vahidAK** [comment]
Question is confusing, at least for me with least experience in statistics. We can make several linear model for each one separately, or using something like (~0, cell_type+organism_part+ age) to set model for all.

**Comment by jafarzadeh91** [comment]
Hi all,

Please have just **one** linear model which includes the mentioned parameters as covariates. 
@vahidAK  I am not sure if I understood your question but as a matter of fact, there is no "E16" here and the age variable is numeric.
@janxue Can you explain a little more about your question? what do you mean by the "cell means" and "reference treatment"?

Bests,
Sina
